### PR TITLE
Prepare standalone docs for publication

### PR DESCRIPTION
**Pull request by cgranade**
Marked as draft until #803 completes.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/566e71e0-9323-4a0b-ba41-de1a71958113_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/install-guide/csinstall.md) | 98 | 91 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/849e88ea-fd85-4aeb-91d4-ce00987453ab_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/install-guide/index.md) | 88 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc1139be-5c29-4d7a-af8a-50eef9db51cd_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/421f722c-b86a-4e60-9019-3ab8ccdac934_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/install-guide/standalone.md) | 88 | 75 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a16873c-b7d4-49bf-9ea1-4182f981f1ac_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/install-guide/toc.yml) | 87 | 26 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9aca613-86eb-48bf-86e0-5fbcffda8a39_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/quickstarts/qrng.md) | 85 | 72 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8a2473e2-5811-4aad-b876-0e5f315084e3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/search.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/quickstarts/search.md) | 81 | 58 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/07c9d2b7-54cc-452a-81f2-6e18609b68bf_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d85dd689aed54a9cc57f0b43d86fe82dcbec819/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28b76b73-a864-4fe2-8d7f-2dfc9ba94e6c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by geduardo** [comment]
We should also update the `articles/install-guide/index.md` file to reflect the new installation guide structure.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/95e2ab3d-4fa5-4190-8c6a-6dec70efc293_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/install-guide/csinstall.md) | 98 | 91 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dfe606fd-7c9b-4603-b4dc-4a824f3d90ff_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/install-guide/index.md) | 88 | 62 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63241e59-be49-4cf6-97c7-ec3e602193ae_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ce7ee8e-3ebc-40af-8b49-daf669623d4a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/install-guide/standalone.md) | 88 | 75 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19c9aa01-0cd1-4e36-8043-524f9d7ff5b8_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/install-guide/toc.yml) | 87 | 26 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c69cf21-8d15-4c97-af91-40f3b6214c78_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/quickstarts/qrng.md) | 85 | 72 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e0b6c6c0-7a1f-4398-897c-835dca0346e1_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/search.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/quickstarts/search.md) | 81 | 58 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b5c99524-aac1-48d5-b7c8-1cceab3b58a1_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0d0d8b7243849994e20adc52ee7d4a8fc6698562/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63496ef4-5fb6-4ada-815d-b5261a623dcd_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/quickstarts/search.md [hunk 1]
```diff
@@ -114,10 +106,11 @@ If you enjoyed this quickstart, check out some of the resources below to learn m
 - [Back to the Getting Started with QDK guide](xref:microsoft.quantum.welcome)
 - Try a more general Grover's search algorithm [sample](https://github.com/microsoft/Quantum/tree/master/samples/algorithms/database-search)
 - [Learn more about Grover's search with the Quantum Katas](xref:microsoft.quantum.overview.katas)
-- Read more about [Amplitude amplification](xref:microsoft.quantum.libraries.standard.algorithms#amplitude-amplification), the quantum computing technique behind Grover's search algorithm
+- Read more about [Amplitude amplification][amplitude-amplification], the quantum computing technique behind Grover's search algorithm
```
Comment (efratshabtai) [inline articles/quickstarts/search.md:109]: Why was the link removed? In the rich view this doesn't show as a clickable link.

Comment (cgranade) [reply articles/quickstarts/search.md:109]: The link was moved to use footnote notation so as to suppress a warning for using fragment identifiers and xrefs together. In previous renderings, the link still worked with no change in target, so it sounds like there was a regression. Let me investigate.

Comment (rmshaffer) [reply articles/quickstarts/search.md:109]: If you look at this file in the [live preview of this PR](https://review.docs.microsoft.com/en-us/quantum/quickstarts/search?branch=pr-en-us-804), the link works correctly.

File: articles/install-guide/index.md [hunk 2]
```diff
@@ -23,6 +23,9 @@ In addition, the QDK provides Q# support for Jupyter Notebooks with the IQ# Jupy
 
 The QDK is available for multiple development environments. Select your preferred setup from the sections below:
 
+- [Q# command line application:](xref:microsoft.quantum.install.standalone) choose this approach if you want to work with Q# from the command line. This does not require a driver or a host program like the below options.
+- [Install Q# for Jupyter Notebooks:](xref:microsoft.quantum.install.jupyter) choose this environment to execute Q# code in cells with embedded text or create quantum computing interactive tutorials. 
+<!---
 - [Install Q# for C#:](xref:microsoft.quantum.install.cs) choose this environment if you want to combine C# and Q# to create a C# host program that calls Q# operations.
+-->
```
Comment (geduardo) [inline articles/install-guide/index.md:30]:
```suggestion
- [Install Q# for .NET:](xref:microsoft.quantum.install.cs) Q# can also be combined with the .NET ecosystem. Choos this environment if you want to combine F# or C# with Q# to create classical host programs that call Q# operations.
```

File: articles/quickstarts/search.md (hunk 1 above)
Comment (cgranade) [reply articles/quickstarts/search.md:109]: Resolution: the link works on the internal preview site and goes to the correct destination, but it does not work correctly in GitHub's built-in rendering.

**Comment by cgranade** [comment]
> We should also update the `articles/install-guide/index.md` file to reflect the new installation guide structure.

Sounds good. Since feature branches are protected, that will need a separate PR either before this that targets feature/standalone, or after this and that targets master.

File: articles/install-guide/index.md (hunk 2 above)
Comment (cgranade) [reply articles/install-guide/index.md:30]: Please see https://github.com/MicrosoftDocs/quantum-docs-pr/pull/806.

Comment (efratshabtai) [reply articles/install-guide/index.md:30]: > Suggested change
>         
>           
>         
>       
>     
>     
>       
>           
>             
>             
>           
>           
>             
>             - [Install Q# for .NET:](xref:microsoft.quantum.install.cs) Q# can also be combined with the .NET ecosystem. Choose this environment if you want to combine F# or C# with Q# to create classical host programs that call Q# operations.

Thank you, I was looking for this.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/015700fa-957e-4a6e-8ef8-20d22d623598_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/install-guide/csinstall.md) | 98 | 91 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0116403-a95d-4fce-80b4-a67166809d64_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/install-guide/index.md) | 90 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ded6b2c3-df8a-4301-a669-8e17b8fba6f9_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9036269-9667-4113-ae69-f99778b4cde2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/install-guide/standalone.md) | 88 | 75 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ecd40bde-363b-4ffa-b628-05de62d70656_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/install-guide/toc.yml) | 87 | 26 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1cda5794-8837-43f7-b3d2-47f6762f4c0c_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/quickstarts/qrng.md) | 85 | 72 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1605651c-940b-4275-bebb-1a8cecabf169_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/search.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/quickstarts/search.md) | 81 | 58 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/569ccba0-cd82-4792-8978-0a57707bb727_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2765b6cff3c28e2911a14734221bcbdf42bcd2ce/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/10e7d9d6-dad4-4718-a88b-4ba37462cec8_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**